transform/snowflake-dbt/models/hightouch/freemium/professional_subscription/freemium_account.sql

```diff
@@ -1,9 +1,10 @@
-{{ config(
-    { "schema": "hightouch",
+{{
+  config({
+    "schema": "hightouch",
     "materialized": "view",
-    "tags" :["hourly","blapi"] }
```

Comment:
does this need "deprecated" as well?

Reply:
In this PR only models used by already paused syncs have been marked as deprecated. For some (like this)  I had to add and remove the tag, as they are used by multiple syncs.

As soon as we make a decision on which other syncs are not needed anymore, a new PR can mark the models as deprecated.